components/homme/src/theta-l/model_init_mod.F90

```diff
@@ -84,6 +84,19 @@ subroutine model_init2(elem,hybrid,deriv,hvcoord,tl,nets,nete )
       call phi_from_eos(hvcoord,elem(ie)%state%phis,&
            temp,elem(ie)%derived%dp_ref,elem(ie)%derived%phi_ref)
       elem(ie)%derived%theta_ref=0
+#endif
+#ifdef HV_REFSTATES_V2
```

Comment:
Does this change affect hydrostatic runs? Is this something for Luca to incorporate in his next merge?

Reply:
yes - but I think this code Luca will still be using the Fortran.